src/audit-trail/interfaces/event.interface.ts

```diff
@@ -1,10 +1,10 @@
-export interface EventAction {
+export interface AuditTrailEventAction {
   object: 'event_action';
   id: string;
   name: string;
 }
 
-export interface Event {
```

Comment:
Do we export these types? If someone were to update the SDK, and they reference `Event` from  their code, would this make their TS build fail? Or start picking up the new `Event` type?

Reply:
Hmm, yeah this would be a breaking change. Perhaps it makes sense to wait to update Audit Trails for the next major update.